### DISCUSSION TITLE
After making media element static, the binding in the xaml to the media element seems to not do anything anymore when navigating to the other page, but still works on the first page and whenever i go back to it, the audio works fine all over its just the UI...anyone any thoughts of what im missing?

### DISCUSSION THREAD

**Discussion by Mashedpoturtles** [CommunityToolkit/Maui#968] [Q&A]

**Comment by bijington** [comment]
Why are you trying to use it as a static shared media element?

**Reply by Mashedpoturtles** [reply]
So that the music that plays on the home page and story player page remains the same and can continue on both pages where it left off, I'm a newbie and that's the only way I know so far to do it

**Reply by bijington** [reply]
It's a bit of an odd concept to share UI across multiple pages like this although I understand what you are trying to solve. I can't tell from the gif but are you playing audio or video in your app?

**Reply by Mashedpoturtles** [reply]
Its fixed now
The comment by jedusei was the answer: https://github.com/dotnet/maui/discussions/13141#discussioncomment-4886443

and yes it was just audio.

**Reply by Mashedpoturtles** [reply]
> It's a bit of an odd concept to share UI across multiple pages like this although I understand what you are trying to solve. I can't tell from the gif but are you playing audio or video in your app?

By the way, why is it an odd concept, how should I have done it? I'm trying to learn from this project so I'd appreciate any teachings you could pass on.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
